### PR TITLE
rebalance: Print human-readable route details with satoshi units and node aliases

### PR DESCRIPTION
**Pull request by ajpwahqgbi**
This PR removes opaque and unintuitive "Sending <amt+fees>msat over <n> hops to rebalance <amt>msat" debug messages and replaces them with non-debug "<n> hops and <fees>sat fees for <amt>sat along route: our alias -> outgoing peer alias -> some node alias -> ... -> incoming peer alias -> our alias".

Salient changes:
- Breaks down fees and rebalance amount separately
- Uses satoshi units instead of harder-to-read msatoshi units
- Shows the attempted route in a human-readable format
- Promotes route attempt messages from 'debug'

### CODE REVIEW

**Comment by ajpwahqgbi** [comment]
FYI it appears CI failed spuriously; at least these jobs seem to have failed with `E: Package 'gcc-4.8' has no installation candidate` which has nothing to do with what I'm doing in this PR.

@cdecker can you please restart the CI jobs?

**Comment by LightningHelper** [comment]
Hi,
I want to test it a bit, but before, can you add an example output, it will make it a bit easier to test.

**Comment by cdecker** [comment]
ACK https://github.com/lightningd/plugins/pull/234/commits/521834887fef795a8f93900555a72d082e015817

**Comment by cdecker** [comment]
ACK https://github.com/lightningd/plugins/pull/234/commits/16dc7ff898c3fe5b1705130ac124d18bfde0257a

File: rebalance/rebalance.py [hunk 1]
```diff
@@ -295,10 +304,20 @@ def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi = Non
...
+                "sent": msatoshi + fees,
+                "received": msatoshi,
+                "fee": fees,
+                "hops": len(route),
+                "outgoing_scid": outgoing_scid,
+                "incoming_scid": incoming_scid,
+                "status": "complete",
+                "message": f"{msatoshi + fees} sent over {len(route)} hops to rebalance {msatoshi}",
+            }
+            midroute_str = reduce(lambda x,y: x + " -> " + y, map(lambda r: get_node_alias(r['id']), route_mid))
```
Comment (LightningHelper) [inline rebalance/rebalance.py:318]: nice

File: rebalance/rebalance.py [hunk 2]
```diff
@@ -295,10 +304,20 @@ def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi = Non
...
+            full_route_str = "%s -> %s -> %s -> %s" % (get_node_alias(my_node_id), get_node_alias(outgoing_node_id), midroute_str, get_node_alias(my_node_id))
+            plugin.log("%d hops and %s fees for %s along route: %s" % (len(route), fees.to_satoshi_str(), msatoshi.to_satoshi_str(), full_route_str))
```
Comment (LightningHelper) [inline rebalance/rebalance.py:320]: This is cool, but wouldn't it be better to also output it in the rpc_result, and not only in the clightning log file?

**Review by LightningHelper — COMMENTED** [review]
The addition is great, but I think it would also be in the rpc output and not only hide in the c-lightning log..

**Comment by cdecker** [comment]
@Mergifyio rebase

**Review by cdecker — APPROVED** [review]

**Comment by cdecker** [comment]
@Mergifyio refresh

**Pull request merged**